src/components/chatMatch/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import cn from 'classnames';
+import useTranslation from 'hooks/useTranslation';
+import smiles from 'assets/smilies.svg';
+
+import './styles.scss';
+
+const ChatMatch = ({ matchs = [] }) => {
+  const t = useTranslation();
+
+  return (
+    <div className="chat-container">
+      <p className="chat-title"> Chats </p>
+      <div className="separator"></div>
```

Comment:
What is this separator supposed to be? I see it only has this properties: 

```
margin-bottom: rem(2px);
width: rem(339px);
```

Maybe you can add that margin to another element